app/controllers/posts_controller.rb

```diff
@@ -1,9 +1,22 @@
 class PostsController < ApplicationController
+  before_action :set_post, only: [:show]
+  before_action :set_user, only: %i[index show]
+
   def index
     @posts = Post.all
   end
 
   def show
+    # @post = Post.find(params[:id])
+  end
```

Comment:
- [ ] Views are not rendering correctly when navigating to `http://[::1]:3000/users/` or to `http://[::1]:3000/users/posts/1`. Similarly, I can see that your corresponding show actions for _user_ and _post_ controllers are empty. Kindly ensure to improve upon the same.

**Views error:**
![image](https://user-images.githubusercontent.com/12745474/198832075-f2dbd213-8a43-4300-aec9-218d4d6c2eeb.png)

Reply:
Hello @rloterh, the views are rendering correctly

this is the correct path for the show http://localhost:3000/users/1/posts/1 [link](http://localhost:3000/users/1/posts/1)

Reply:
Hi @Roseokpe 

My comments above was actually pointing to  `http://localhost:3000/users/1/posts/1`

Kindly check the below screenshot for the same:
![image](https://user-images.githubusercontent.com/12745474/198833662-f15ce878-06ce-48c4-b457-0c0fcb7135b1.png)

If the version on your local computer works correctly, I suggest that you update the repository version accordingly.

Cheers! 😃